Completed work section of the mobile first portfolio project

FevenSeyfu opened the pull request.

FevenSeyfu committed:
Completed work section

huseyinbiyik commented:
Hi @FevenSeyfu 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no Pull Request description. Please add the changes made on this branch to the PR description.
<img width="859" alt="Screenshot 2022-12-03 at 12 30 39 AM" src="https://user-images.githubusercontent.com/25887253/205391239-b6e37b23-a60c-4002-a99a-df917f667402.png">

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

huseyinbiyik commented:
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no PR description. Kindly check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Write%20a%20meaningful%20title%20for%20your%20pull%20request%20and%20always%20add%20a%20short%20summary%20of%20changes%20introduced%20in%20the%20pull%20request.) to learn about PR description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

FevenSeyfu commented:
Completed the previous works section for my portfolio project.
What has been added,

- Added my previous work demo section
- Added my previous work description
- Styled the layout with grid and flexbox combines

Yerimah commented on style.css:
```diff
@@ -83,3 +83,146 @@ ul{
+    line-height: 20px;
+    line-height: 110%;
+}
+.main-btn{
+    margin: 1rem 0 0.75rem 0;
+    background-color: #FF6B00;
+    color: white;
+    width: 124px; 
+    height: 48px;
+    font-family: 'Inter', sans-serif;
```
- [x] Hey 👋,  I would suggest you style the button with `class="main-btn"` properly to make your design as close to the Figma design template as much as possible.

> Your design
![your-btns-design](https://user-images.githubusercontent.com/71140133/205410305-1c5f62a0-f737-4d28-a7b7-961c005b2bdb.PNG)

> Figma Design Template
![btns-sect](https://user-images.githubusercontent.com/71140133/205410342-8c4e505a-416b-40c0-b1d2-b143df08845f.PNG)

- [x] I would also suggest that you implement the hover, active and pressed efffect on your buttons to make your design look like the Figma design. A look at the design guide in Figma will be helpful in implementing the colors of the `hover`, `active`  and `pressed` effect.

> Button design Guide
![btns](https://user-images.githubusercontent.com/71140133/205410491-2bade3d6-0b26-4523-b55a-694e5ac8094f.PNG)

FevenSeyfu replied:
Thank you for the feedback. I have removed the border and matched the style of button with the Figma design. I have also included added hover and active states.

Yerimah commented on style.css:
```diff
@@ -83,3 +83,146 @@ ul{
+    color: white;
+    border: none;
+}
+.btn-max-width{
+    width: 100%;
+    background-color: #FF6B00;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
```
- [x] Make sure to always follow the button styling in the Figma design template for all the cards in the work section. You must stick to the design as much as possible (e.g., font, colors, images, text, margins). Adjust the height of the buttons with `class="btn-max-width"` .

> Figma Design Template
![fig-card-dsn](https://user-images.githubusercontent.com/71140133/205411990-323656f9-1977-4ac9-8e56-ae0ece2fd9a3.PNG)

> Your Design
![card-btns](https://user-images.githubusercontent.com/71140133/205412004-43e3ef10-72b0-41a4-8da4-55c888318b08.PNG)

FevenSeyfu replied:
Thank you for your feedback. I have changed the styling of the button specifically the height to match the Figma design.

Yerimah reviewed (changes requested):
# STATUS: REQUESTED CHANGES 🔴 

Hi @FevenSeyfu 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4b70556d45436b5038554458344c37536b552f67697068792e676966](https://user-images.githubusercontent.com/71140133/205412270-f60c56b7-899d-48e1-9bfb-a75cc0d64218.gif)

## Highlights
- No linter errors ✔️ 
- Github flow ✔️ 
- Descriptive PR title and summary ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FevenSeyfu committed:
Changed Style on main button

FevenSeyfu commented:
Changed the style to match the Figma design as below:

- removed the border from the button to match Figma design,
- Changed the height of the action button,
- added hover and active styles on the button.

Lizdev-05 reviewed (approved):
Hi @FevenSeyfu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media1.giphy.com/media/26gsvAm8UPaczzXz2/giphy.gif"/>
## Highlights
- No linter errors ✔️
- Github flow ✔️
- Descriptive PR title and summary ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
